capstone/src/main/java/com/google/univiz/SuggestionData.java

```diff
@@ -1,4 +1,18 @@
 package com.google.univiz;
 
+import com.google.auto.value.AutoValue;
+import com.google.gson.annotations.SerializedName;
+import com.ryanharter.auto.value.gson.GenerateTypeAdapter;
+import javax.annotation.Nullable;
+
 /** SuggestionData represents the college data specific to the Search feature */
-public class SuggestionData {}
+@GenerateTypeAdapter
+@AutoValue
+abstract class SuggestionData {
+  @SerializedName("school.name")
```

Comment:
Just noting that there's not a need to provide `@SerliazedName` unless you want to change the [default mapping](https://github.com/rharter/auto-value-gson#usage).

This comment applies to all usages of @SerializedName in this PR.

Reply:
updated! I didn't need the SerializedName for id :)